docs/src/index.md

```diff
@@ -9,7 +9,13 @@ For information about ChainRules, including how to write rules, refer to the gen
 [![](https://img.shields.io/badge/docs-main-blue.svg)](https://JuliaDiff.github.io/ChainRulesCore.jl/dev)
 [![](https://img.shields.io/badge/docs-stable-blue.svg)](https://JuliaDiff.github.io/ChainRulesCore.jl/stable)
 
-## Canonical example
+## Testing Method Table Sensibility
+A basic feature of ChainRulesTestUtils is it's ability to check that the method table for `rrule` and `frule` remains sensible.
+This searches the method table for methods that should not exist and when it fails tells you where they were defined.
```

Comment:
let's be consistent here and throughout on if we're talking about "the method table" (singular) or "the method tables (for `frule` and `rrule`)" (plural). I'm not sure which is more accurate but i think consistency helps to just even follow what's being said :)

since  the function itself is "`test_method_tables_sensibility`" i guess we're going with plural:
```suggestion
A basic feature of ChainRulesTestUtils is its ability to check that the method tables for `rrule` and `frule` remain sensible.
This searches the method tables for methods that should not exist and when it fails tells you where they were defined.
```